lib/main.js

```diff
@@ -63,10 +63,10 @@ exports.activate = function () {
   }
 }
 
-const STYLED_REGEX = /\bstyled\b/i
+const CSS_REGEX = /\bstyled\b|\bcss\b/i
 
-function languageStringForTemplateTag (tag) {
-  if (STYLED_REGEX.test(tag)) {
+function languageStringForTemplateTag(tag) {
```

Comment:
At the risk of sounding nitpicky, can you add back the space before the open parentheses? That's part of the code style across Atom repos. Thanks!

Reply:
@daviwil I thought all repos had linters as part of the tests?

Reply:
I only see `coffeelint` hooked up on this repo, must not have had `standard` added to it once JavaScript code was introduced.

Reply:
no problemo

Reply:
atom-ide-ui has a bad habit of applying formatting on save, regardless of eslint